IDEAS/Buildings/Components/RectangularZoneTemplate.mo

```diff
@@ -46,6 +46,8 @@ model RectangularZoneTemplate
   parameter Boolean hasWinCei = false
     "Modelling window for ceiling if true"
     annotation(Dialog(tab="Ceiling", group="Window details", enable=not (bouTypCei == IDEAS.Buildings.Components.Interfaces.BoundaryType.None)));
+  parameter Boolean hasIntZone = false "True if the zone contains an internal wall with both faces connected to itself"
```

Comment:
Rephrasing this since the faces are not connected to itself. They are connected to the zone. Also, putting comment on a newline.

Reply:
Renaming `intZone` since this can be read as `internal zone`, which does not make sense. I'd replace it by `intWal` or `int`. Since `intWal` emphasizes 'wall' more than `int`, I prefer `int` since the point of this model is that the internal wall is internal in the zone.